src/aap_eda/core/models/source.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright 2022 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import uuid
+
+from django.db import models
+
+from aap_eda.core.enums import RestartPolicy
+
+__all__ = "Source"
+
+
+class Source(models.Model):
```

Comment:
if we are going to replicate the activation model, we need too the is_enabled field.

Reply:
By default is_enabled=True we don't have a place to start activation for the source, we have to start it right away. It will eventually become Activation eventually.

Reply:
ok.